src/main/java/adris/altoclef/AltoClefCommands.java

```diff
@@ -6,6 +6,7 @@
 import adris.altoclef.commandsystem.CommandList;
 
 /// This structure was copied from a C# project. Fuck java. All my homies hate java.
+// ^ cringe
```

Comment:
I'm totally with you on this one, but please remove this comment lol

Feel free to make a github issue, if you do I'll seriously consider removing this line.

Reply:
made an issue & removed the comment